File: C#/백준/Bronze/5988. 홀수일까 짝수일까/홀수일까 짝수일까.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                String b = Console.ReadLine();
                if ((int)b[b.Length-1]%2==0){
                    Console.WriteLine("even");
                }
                else{
                    Console.WriteLine("odd");
                }
            }
        }
    }
}

File: Java/백준/Bronze/5988. 홀수일까 짝수일까/홀수일까 짝수일까.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i=0;i<a;i++){
            String s = in.next();
            if ((int)s.charAt(s.length()-1)%2==0){
                System.out.println("even");
            }
            else{
                System.out.println("odd");
            }
        }
        
    }
}